enterprise_subsidy/apps/api/v1/views/content_metadata.py

```diff
@@ -99,7 +100,7 @@ def get(self, request, content_identifier, enterprise_customer_uuid):
                 enterprise_customer_uuid[0],
                 content_identifier
             )
-            content_summary = catalog_client.summary_data_for_content(content_data)
+            content_summary = summary_data_for_content(content_data)
```

Comment:
Why not keep this a catalog_client method?  All other usages of content_metadata helper functions are hidden behind catalog_client, this seems to deviate from that pattern.

Reply:
i believe this clarifies the separation of concerns - allowing the catalog client to mostly just be how we interact with the catalog service, and the new metadata api focus on how subsidy service is going to process/transform the catalog data into what it needs - how does that sound?

Reply:
That idea sounds good to me, but it feels like this PR only partially commits to that idea because some processing of the metadata is still happening inside the catalog_client (extracting the price).  It would feel more like concerns have been separated if catalog_client exposed more of a 1:1 of class methods to API endpoints.